disnake/guild.py

```diff
@@ -2163,11 +2180,19 @@ async def create_scheduled_event(
 
         Creates a :class:`GuildScheduledEvent`.
 
-        If ``entity_type`` is :class:`GuildScheduledEventEntityType.external`:
+        Based on the channel/entity type, there are different restrictions regarding
+        other parameter values, as shown in this table:
+
 
-        - ``channel`` should not be set
-        - ``entity_metadata`` with a location field must be provided
-        - ``scheduled_end_time`` must be provided
+        .. csv-table::
+            :widths: 25, 30, 20, 25
+            :header: "``channel``", "``entity_type``", "``scheduled_end_time``", "``entity_metadata`` with location"
+
+            :class:`VoiceChannel`, :attr:`~GuildScheduledEventEntityType.voice` (set automatically), optional, unset
+            :class:`StageChannel`, :attr:`~GuildScheduledEventEntityType.stage_instance` (set automatically), optional, unset
```

Comment:
these are slightly inaccurate, and should refer to the type attribute on the object instead. Passing a PartialMessageable with the voice channel type would work, for example. Passing an `Object` with the type attribute set to `ChannelType.voice` would also work. The class doesn't actually matter, and this distinction is important.

Reply:
I can see it being a bit confusing to users, hopefully my wording isnt too rough

Reply:
I don't think so.